### DISCUSSION TITLE
Assets vs Ops

### DISCUSSION THREAD

**Discussion by bernardocortez** [dagster-io/dagster#18864] [Q&A] [answered]
Hi there.
We are using jobs and ops as a basis for the architecture of our code. I am observing that the recent release are using assets more and more, in what seems to be a shift from previous versions.

What exactly is the difference between assets and ops?

**Comment by smackesey** [comment, marked as answer]
Hi @bernardocortez, I recommend looking at our docs-- start here: https://docs.dagster.io/guides/dagster/how-assets-relate-to-ops-and-graphs

In general we are most focused on assets, so I recommend using them whenever possible.
